packages/core/package.json

```diff
@@ -11,6 +11,7 @@
   "scripts": {
     "generate:schema": "tsx src/server/generator/generateGraphQLSchemaFile.ts",
     "generate:codegen": "graphql-codegen",
+    "generate:copy-back": "copyfiles \"@generated/**/*\" $DESTINATION",
```

Comment:
Out of curiosity, why do we need to copy back?

Reply:
To have proper typings during development. The merger happens on the `.faststore` folder, but the store imports types and the `gql` helper from `@faststore/core`. We copy back from `.faststore` to `@faststore/core` so the types from the merger (core + store) are served from `@faststore/core`.